package/plugin-gargoyle-openvpn/files/www/js/openvpn.js

```diff
@@ -214,12 +214,13 @@ function saveChanges()
 
 		// if anything in server section or client section has changed, restart openvpn
 		// otherwise we're just adding client certs to a server and restart shouldn't be needed
+		// unless we are adding/removing a client with a routed subnet, then we should give openvpn a kick to push the server routes
 		var openvpnCurrentlyRunning = (tunIp != "" && openvpnProc != "" && (remotePing != "" || openvpnConfig == "server"))
 		if(openvpnConfig == "disabled")
 		{
 			commands = "/etc/init.d/openvpn stop ; " + commands
 		}
-		else if(commands.match(/uci.*openvpn_gargoyle\.server\./) || openvpnConfig == "client" || (!openvpnCurrentlyRunning) )
+		else if(commands.match(/uci.*openvpn_gargoyle\.server\./) || commands.match(/uci.*openvpn_gargoyle\.[^server].*subnet_ip/) || openvpnConfig == "client" || (!openvpnCurrentlyRunning) )
```

Comment:
although i now remark that perhaps the [^server] check is unnecessary and would fix the corner case... maybe fix when the complaints roll in